### PR TITLE
Arm code clean up

### PR DESCRIPTION
**Pull request by NoahWeishan**

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java
Comment (python-42) [inline src/main/java/frc/robot/Constants.java:39]: Why did you change this?

File: src/main/java/frc/robot/arm/Arm.java
Comment (python-42) [inline src/main/java/frc/robot/arm/Arm.java:38]: Delete the commented out lines

File: src/main/java/frc/robot/arm/Arm.java
Comment (python-42) [inline src/main/java/frc/robot/arm/Arm.java:59]: This functionality needs to continue to exist so that the slow shooting for amp scoring is triggered properly. 

You can do this by checking the target angle against the constant and isOnTarget

**Review by python-42 — CHANGES_REQUESTED** [review]
Good job working through the code review feedback. Just a couple of changes and this can be merged.

**Review by python-42 — APPROVED** [review]
DO NOT MERGE THIS
Approved pending robot testing.

**Review by pordonj — APPROVED** [review]

**Pull request merged**
